couchdb/docker-entrypoint.sh

```diff
@@ -96,6 +96,8 @@ if [ "$1" = '/opt/couchdb/bin/couchdb' ]; then
     if [ "$COUCHDB_LOG_LEVEL" ]; then
         if ! grep -Pzoqr "\[log\]\nlevel =" /opt/couchdb/etc/local.d/*.ini; then
             printf "\n[log]\nlevel = %s\n" "$COUCHDB_LOG_LEVEL" >> $CLUSTER_CREDENTIALS
+        else
+            sed -i "s/level = .*/level = $COUCHDB_LOG_LEVEL/g" $CLUSTER_CREDENTIALS
```

Comment:
In a happy path search, this would find `\[log\]\nlevel =` in `CLUSTER_CREDENTIALS` which we define [line 15](https://github.com/medic/cht-core/blob/27d3446fc631fb4e111d2ebab12f52e635c32ebb/couchdb/docker-entrypoint.sh#LL15C22-L15C70) as `/opt/couchdb/etc/local.d/cluster-credentials.ini`. In this case it is fine to naively replace the `level` value in  `cluster-credentials.ini`.

However, is there a slightly unhappy path where `\[log\]\nlevel =` is found in `/opt/couchdb/etc/local.d/deployment-specific-credentials.ini`?  We would then attempt, and fail, to replace in `cluster-credentials.ini`.  Thus line 100's `sed`  would have no affect.

Another way of looking this might be: if we only ever set `level` in  `cluster-credentials.ini`, we could remove the wildcard search in line 97 and tightly bind it to `cluster-credentials.ini`.

Reply:
Such an eagle eye, @mrjones-plip. Love it. 

I'm tempted to keep it simple and change the `if` to only search in `cluster-credentials.ini`. Deferring to @Hareet since he is most familiar with this code.

Reply:
@Hareet could you please provide some insights about this, please?

Reply:
Cool finds. I like: 
`if to only search in cluster-credentials.ini`, no need to use wildcard search

Reply:
Thanks for the input, @Hareet ! I've updated all the ini search functions to only target `cluster-credentials.ini` for consistency. Would you mind a proper review? Thanks!

Reply:
much cleaner - thanks!